Consul.Test/ACLTest.cs

```diff
@@ -30,6 +30,9 @@ public class ACLTest : BaseFixture
         public async Task ACL_CreateDestroyLegacyToken()
         {
             Skip.If(string.IsNullOrEmpty(TestHelper.MasterToken));
+            var info = await _client.Agent.Self();
+            var version = new Version(info.Response["Config"]["Version"]);
+            Skip.If(version >= new Version("1.11.0"), "Legacy ACL system was removed in Consul 1.11.");
```

Comment:
I have done this in the same way we skip tests in `AuthMethodTest.cs`, where it's repeated in each test:

https://github.com/G-Research/consuldotnet/blob/9f874844b42ced7c971447c0a461abe66e25d911/Consul.Test/AuthMethodTest.cs#L32-L33

I do wonder if this might instead live at the class level, or in `TestHelper.cs`. Thoughts?

Reply:
I think that the best would be to create new method in ACLTest class to avoid code duplication.

e.g:
```
  public class ACLTest : BaseFixture
  {
        async Task SkipIfAclNotSupportedAsync()
        {
            var info = await _client.Agent.Self();
            var version = new Version(info.Response["Config"]["Version"]);
            Skip.If(version >= new Version("1.11.0"), "Legacy ACL system was removed in Consul 1.11.");
        }

        [SkippableFact]
        public async Task ACL_CreateDestroyLegacyToken()
        {
            Skip.If(string.IsNullOrEmpty(TestHelper.MasterToken));
            await SkipIfAclNotSupportedAsync();
...
```

PS:

Reply:
Agreed 👍
Implemented in 1cf9460.